### DISCUSSION TITLE
cors error

### DISCUSSION THREAD

**Discussion by Mobivs** [FlowiseAI/Flowise#157] [Q&A] [answered]
Has anyone has encountered cors errors upon calling the api?
If you have, can you share what you did about it?
I'm running on render.com

**Comment by Mobivs** [comment, marked as answer]
Got it. I had to add
"Access-Control-Allow-Origin": "*"
to the client.js file.

**Reply by mejaatg** [reply]
can you please tell me, where you found the client.js file????
